src/types.ts

```diff
@@ -1,4 +1,5 @@
 import type { ISpectralDiagnostic, IFunctionResult } from '@stoplight/spectral-core';
+import { JSONSchema7 }from "json-schema"
```

Comment:
```suggestion
import type { JSONSchema7 }from "json-schema"
```

Reply:
It's not resolved.

Reply:
Sorry, didn't commit. It is now, please check again.